persistence-api/src/main/java/io/stargate/db/cdc/SchemaAwareCDCProducer.java

```diff
@@ -61,7 +60,7 @@
   public CompletableFuture<Void> publish(MutationEvent mutation) {
     TableSchemaManager schemaManager =
         tableSchemaManager.computeIfAbsent(
-            mutation.getTable().getId(), k -> new TableSchemaManager(this::createTableSchemaAsync));
+            mutation.getTable(), k -> new TableSchemaManager(this::createTableSchemaAsync));
```

Comment:
Just wondering: if we get several schema changes in quick succession, how do we guarantee that the producer makes its schema changes in the same order? Does it matter?

Reply:
We will use the `Schema` that is volatile and swapped each time there's a change, so it can cause race conditions when dealing with schema changes and mutations.

I think ordering doesn't matter though. In the CDC service, each mutation will get a table schema "version". The same will be occur on the producer implementation (kafka/pulsar): each mutation event record will have a corresponding schema version, more than one version can coexist in schema registries.

Reply:
The ordering doesn't matter as long as changes are backward compatibility, isn't it?